packages/react-slider/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Fluent UI React Slider component.",
   "main": "lib-commonjs/index.js",
   "module": "lib/index.js",
-  "private": "true",
+  "private": "false",
```

Comment:
nit: This can be removed, it is the same without this line and afaik other published components do not include this.